include/CoreLocation/CLHeading.h

```diff
@@ -26,11 +27,12 @@ typedef double CLHeadingComponentValue;
 
 CORELOCATION_EXPORT_CLASS
 @interface CLHeading : NSObject <NSCopying, NSSecureCoding>
-@property (readonly, nonatomic) CLLocationDirection magneticHeading STUB_PROPERTY;
-@property (readonly, nonatomic) CLLocationDirection trueHeading STUB_PROPERTY;
-@property (readonly, nonatomic) CLLocationDirection headingAccuracy STUB_PROPERTY;
-@property (readonly, copy, nonatomic) NSDate* timestamp STUB_PROPERTY;
-@property (readonly, nonatomic) CLHeadingComponentValue x STUB_PROPERTY;
-@property (readonly, nonatomic) CLHeadingComponentValue y STUB_PROPERTY;
-@property (readonly, nonatomic) CLHeadingComponentValue z STUB_PROPERTY;
+@property (readonly, nonatomic) CLLocationDirection magneticHeading;
+@property (readonly, nonatomic) CLLocationDirection trueHeading;
+@property (readonly, nonatomic) CLLocationDirection headingAccuracy;
+@property (readonly, copy, nonatomic) NSDate* timestamp;
+@property (readonly, copy, nonatomic) NSString* description;
+@property (readonly, nonatomic) CLHeadingComponentValue x;
+@property (readonly, nonatomic) CLHeadingComponentValue y;
+@property (readonly, nonatomic) CLHeadingComponentValue z;
```

Comment:
Yes, I unstubbed all these properties together but the Windows APIs never actually give any kind of x, y, or z values so I never used them. Restubbed.